bindings/protocols/coap/index.html

```diff
@@ -499,6 +540,16 @@ <h3>Form terms</h3>
                             <a href="http://www.w3.org/2001/XMLSchema#string">string</a> (e.g., <code>GET</code>, <code>PUT</code>, <code>POST</code>, <code>DELETE</code>, <code>FETCH</code>, <code>iPATCH</code>, or <code>PATCH</code>)<p>(See the <a href="https://www.iana.org/assignments/core-parameters/#method-codes">CoAP Method Codes</a> registry for the set of possible values.)</p>
                         </td>
                     </tr>
+                    <tr>
+                        <td><code>cov:minPollingInterval</code></td>
+                        <td>
+                            Provides a hint of what minimum polling interval (in seconds) the server is willing to accept before rejecting the client with a 4.29 (Too Many Requests) response [[RFC8516]].
+                        </td>
+                        <td>optional</td>
+                        <td>
+                            <a href="http://www.w3.org/2001/XMLSchema#unsignedInt">unsignedInt</a>
```

Comment:
No objection. I just want to mention that the possible range is rather large. Maybe `unsignedShort` is sufficient but I don't know whether this really matters in this case..

Reply:
According to [RFC 8516](https://www.rfc-editor.org/rfc/rfc8516.html#section-3), which defines the 4.29 (Too Many Requests) response code, "the Max-Age option is used to indicate the number of seconds after which the server assumes it is OK for the client to retry the request". And, according to [RFC 7252](https://datatracker.ietf.org/doc/html/rfc7252#section-5.10.5), "the option value is an integer number of seconds between 0 and 2**32-1 inclusive (about 136.1 years)."

So, yes, the range is rather large. But that's how CoAP defines it. Since we want to preempt the server's 4.29 error response by specifying the interval in the TD, I think we should go for the same range here.